spec/game_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_relative '../game'
+
+describe Game do
+  before :each do
+    @new_game = Game.new('2010/09/04', multiplayer: true)
+  end
+
+  it 'multiplayer should return true' do
+    multiplayer = @new_game.multiplayer
+    expect(multiplayer).to be_truthy
+  end
+
+  it 'date should return 2010/09/04' do
+    date = @new_game.last_played_at
+    expect(date).to eql('2010/09/04')
+  end
+
+  it 'can_be_archived? should return true' do
+    check = @new_game.can_be_archived?
+    expect(check).to be_truthy
+  end
+
+  it 'Game should be a class of Item' do
+    expect(@new_game).to be_kind_of(Item)
+  end
+end
```

Comment:
- [ ] Some of the tests are failing please make sure that is passing in order to complete the requirements 
<img width="1256" alt="Screenshot 2023-01-11 at 14 44 32" src="https://user-images.githubusercontent.com/61790587/211823020-e22f0450-e2db-4692-a7b9-01a9b980311b.png">